.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -31,6 +29,26 @@ body:
         Example: When using SSH to connect to my remote Linux machine from my local macOS device in Ghostty, I try to run `clear`, and the screen does not clear.  Instead, I see the following error message printed to the terminal: `Error opening terminal: xterm-ghostty.`
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: |
+        Provide a summary of how you expect Ghostty to behave in this situation.  Include any relevant documentation links.
+      placeholder: |
+        Example: When I run `clear`, I expect the screen to be cleared and the prompt to be redrawn at the top of the window.
```

```suggestion
        The screen to be cleared and the prompt is redrawn at the top of the window.
```
You don't add “Example:” anywhere else, and in my experience most people don't word these fields like that.

---

https://github.com/ghostty-org/ghostty/blob/b6f338065eb45935627a51a7d9102019aaeae1c0/.github/DISCUSSION_TEMPLATE/issue-triage.yml?plain=1#L31-L32

We actually used that same format initially.  I'll be happy to change all 3 fields, I agree it is clunky

---

Ah, missed that.  It should probably be removed since it's kinda obvious that it's an example.